ring_doorbell/__init__.py

```diff
@@ -90,18 +100,20 @@ def _process_cached_session(self):
     def _get_oauth_token(self):
```

Comment:
In a future PR we should consider moving this logic out of this function as well. Separate the API logic from the Auth logic.

Reply:
In Home Assistant, the idea will be to no longer store username/password. It's only used on initial set up, after that we will just store the refresh/access tokens.

Reply:
Is this for a future version, or already implemented in the 103.5 version? I.e. do I still need to store the Ring password in HA, or is that not needed anymore?

Reply:
It is still required to put the username and password into your HA configuration